C/162_Find_Peak_Element.cpp

```diff
@@ -0,0 +1,55 @@
+//A peak element is an element that is greater than its neighbors.
```

Comment:
could you change the format according to the Two Sum committed by forest5013? Let's all use the some format.